discord/client.go

```diff
@@ -611,6 +612,12 @@ func limitRPCs(ctx context.Context, rpcs <-chan rpc) {
 			}
 		retry:
 			r, err := http.DefaultClient.Do(rpc.req.WithContext(ctx))
+			if err == nil && r.StatusCode == http.StatusInternalServerError && errRetries < 3 {
+				time.Sleep(500 * time.Millisecond)
```

Comment:
Should we just go ahead and add a backoff here?

Reply:
Done.